### PR TITLE
[857 Request] Refactor  changes to unify methods with utility paths as params

### PR DESCRIPTION
**Pull request by genslein**
https://github.com/wal-g/wal-g/pull/858

This was the follow-up refactor to merge the code paths.  Will make additional changes as requested here.

### CODE REVIEW

File: cmd/fdb/backup_list.go [hunk 1]
```diff
@@ -16,7 +17,7 @@ var backupListCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		folder, err := internal.ConfigureFolder()
 		tracelog.ErrorLogger.FatalOnError(err)
-		internal.DefaultHandleBackupList(folder)
+		internal.DefaultHandleBackupList(folder, utility.BaseBackupPath)
```
Comment (usernamedt) [inline cmd/fdb/backup_list.go:20]: Maybe I missed that point, but why we can't instead call this function like this?
```
internal.DefaultHandleBackupList(folder.GetSubFolder(utility.BaseBackupPath))
```

Comment (genslein) [reply cmd/fdb/backup_list.go:20]: Ah fair point. I'll update

**Comment by usernamedt** [comment]
Looks like the `bin/golangci-lint` is uploaded unintentionally

**Review by usernamedt — APPROVED** [review]
lgtm, remove the binary file

**Comment by genslein** [comment]
> lgtm, remove the binary file

Ah, I thought I ignored it. Removed

**Pull request merged**